### PR TITLE
Display logo in Payment Options during Checkout

### PR DESCRIPTION
**Pull request by seherkdc**
This resolves issue #82

### CODE REVIEW

File: view/frontend/web/template/payment/razorpay-form.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
                name="payment[method]"
                class="radio"
                data-bind="attr: {'id': getCode()}, value: getCode(), checked: isChecked, click: selectPaymentMethod, visible: isRadioButtonVisible()"/>
-        <label data-bind="attr: {'for': getCode()}" class="label"><span data-bind="text: getTitle()"></span></label>
```
Comment (ccrims0n) [inline view/frontend/web/template/payment/razorpay-form.html:7]: @seherkdc Why have we removed this?

Comment (seherkdc) [reply view/frontend/web/template/payment/razorpay-form.html:7]: @ccrims0n The removed line of code displayed "Razorpay" in text. If we're adding the Razorpay wordmark logo, then both text and image become repetitive.

Comment (ccrims0n) [reply view/frontend/web/template/payment/razorpay-form.html:7]: @seherkdc Can you please share a screenshot of the updated view.

Comment (seherkdc) [reply view/frontend/web/template/payment/razorpay-form.html:7]: Screenshot of only Razorpay Logo
Screenshot (1)


Screenshot of Razorpay text with Logo
Screenshot

Comment (ccrims0n) [reply view/frontend/web/template/payment/razorpay-form.html:7]: looks good.

**Review by ccrims0n — APPROVED** [review]

**Pull request merged**